elixir/apps/domain/lib/domain/config/definitions.ex

```diff
@@ -598,4 +598,9 @@ defmodule Domain.Config.Definitions do
   Boolean flag to turn Multi-Site resources functionality on/off for all accounts.
   """
   defconfig(:feature_multi_site_resources_enabled, :boolean, default: false)
+
+  @doc """
+  Boolean flag to turn API Client UI functionality on/off for all accounts.
+  """
+  defconfig(:feature_api_client_ui_enabled, :boolean, default: false)
```

Comment:
Let's use one flag for all REST API stuff:
```suggestion
  defconfig(:feature_rest_api_enabled, :boolean, default: false)
```

Reply:
Sounds good 👍

Reply:
updated.